package.json

```diff
@@ -53,10 +58,17 @@
     "build:esm": "yarn tsc -m es6 --outdir dist/esm",
     "clean": "rm -rf dist",
     "lint": "eslint '{src,test}/**/*.ts'",
-    "test": "nyc mocha 'test/**/*.test.{ts,tsx}'"
+    "test": "nyc mocha 'test/**/*.test.{ts,tsx}'",
+    "prepare": "husky install",
+    "commit": "git add . && cz"
```

Comment:
If you want to make a new commit, run `yarn commit` and you'll be walked step by step